Packs/Vectra_AI/pack_metadata.json

```diff
@@ -3,16 +3,26 @@
     "categories": [
         "Network Security"
     ],
-    "currentVersion": "1.0.0",
     "created": "2022-06-27T10:00:00Z",
-    "description": "This content pack allows to create incidents based on Vectra Accounts/Detections/Hosts objects.",
+    "currentVersion": "1.1.0",
+    "dependencies": {
+        "CommonTypes": {
+            "display_name": "Common Types",
+            "mandatory": true
+        }
+    },
```

Comment:
```suggestion
```
Due to my dependencies answer on the other comment, you can delete that.

Reply:
Done as suggested